analysis/topeft_run2/make_cr_and_sr_plots.py

```diff
@@ -912,20 +1015,31 @@ def make_all_sr_data_mc_plots(dict_of_hists,year,save_dir_path):
 
     # Loop over hists and make plots
     skip_lst = ['ptz', 'njets'] # Skip this hist
+
+    #CHANGE: a separate dict to just include sumw2 hist
```

Comment:
Remove

Reply:
I have edited this line to be a comment.